libs/ml/tests/unit/model/dnn_regressor_test.cpp

```diff
@@ -106,7 +106,7 @@ TYPED_TEST(DNNRegressorModelTest, adagrad_dnnregressor)
 {
   using DataType = typename TypeParam::Type;
   ASSERT_TRUE(regressor_details::RunTest<TypeParam>(fetch::ml::OptimiserType::ADAGRAD,
-                                                    DataType{1e-4f}, DataType{0.03f}, 400));
+                                                    DataType{1e-4f}, DataType{0.04f}, 400));
```

Comment:
Ditch scientific notation, "0.0001"

Reply:
these DO need string constructors! Type<DataType>("0.0001")

Reply:
The ML stuff shouldn't be in there! It's been pulled in from another PR.

Reply:
To avoid duplicating work I'm inclined to fix these in my defloating ml PR. This PR (for math) will go in first so it won't overwrite the ml one.